Source/Mosa.Korlib/System/Boolean.cs

```diff
@@ -53,5 +54,46 @@ public override string ToString()
 		{
 			return m_value ? "True" : "False";
 		}
+
+		// IComparable interface
+		//
+		public override bool Equals(object obj)
+		{
+			if (this.GetType() == obj.GetType() && obj is Boolean)
```

Comment:
Is the explicit type check required? `is` should already take care of that, no?

Reply:
@copygirl yes, you're right, explicit type check is not required, it doesn't hurt either. `is` alone should be enough. I'll correct this when I get back to home tonight.